analysis_data_preprocess/create_PminusE_GPCP-OAFlux_timeseries.py

```diff
@@ -1,61 +1,66 @@
-#This script is to combine precipication from GPCP and evaporation from OAFlux to derive P minus E variable
-#Generated Oct25th by Jill Zhang 
-
-import cdms2 
-import os
-import numpy
+# This script is to combine precipication from GPCP and evaporation from OAFlux to derive P minus E variable
```

Comment:
Why doesn't this one start with `# flake8: noqa` when some of the others in this directory do?

Reply:
Added # flake8: noqa to the file, good catch